:sparkles: Allow to store layer's expression filters (authenticathed users)

wlorenzetti opened the pull request:
Closes: #622
Closes: #624

Commit:
Fix 'mode=delete' behaviour.

Commit:
Add FilterLayerSaved model.

Commit:
Add mode 'save' and 'delete_saved' in response_filtertoken.

Raruto commented:
@wlorenzetti I changed the title and kept only the `feature` tag (just for simplicity, otherwise it becomes a bit doubtful, eg. within the release page).

Commit:
Add FilterLayerSaved to django admin.

Commit:
Add filtertoken new workflow with mode 'save' 'delete_saved' 'apply'

Commit:
Testing.

Commit:
Testing.

Commit:
Testing.

Commit:
Testing.

Commit:
Refactoring and documentation.

Commit:
Add 'filters' property to 'layers' of /api/config REST API.

Commit:
Fix

Commit:
Change unique condition fro FilterLayerSaved.

Commit:
Add test fo /api/config REST API.

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/43226d3073dcb9a1611cac968f46027c43ce5abe

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/001b107ec582fdde296dfd0c91073a8b14d1b9ef

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/84421d7131987db18965a8a4b8675c189d15c559

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/ffbb2a876d419bf568e85776f220cd43ea973585

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/70c6dbbf201659aa5f505ae6b7a75ed7d7e9af6f

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/358886cd2085387b780668eb9645fe35d2d99862

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/de741892fd1ae81ad5f2260c6f3521128f2b60fc

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/27e043fb2d52d488f76a3db0551cf88e2d50e93f

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/9e404cbbd7ddab7137bdc635f60b56b3456e1fec
    g3w-client-plugin-editing: https://github.com/g3w-suite/g3w-client-plugin-editing/pull/71/commits/17685abba5518f18caf1bc67b15de48d2fcf20a5

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/526a8ea0b61818d4e7ad9cc7f86487ee7899cec9

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/d3b90db092e256ef54859277cfb4b5f8258fb1aa

volterra79 committed:
:sparkles: Client:
    g3w-client: https://github.com/g3w-suite/g3w-client/pull/503/commits/533681dbc106eb7d94bc24d7c60e36f00fa3a0a7

Commit:
Merge branch 'dev' into Fix_issue_622

# Conflicts:
#	g3w-admin/client/static/client/css/app.min.css
#	g3w-admin/client/static/client/js/app.min.js
#	g3w-admin/editing/static/editing/js/plugin.js
